integrations/haproxy/documentation.yaml

```diff
@@ -74,13 +40,16 @@ podmonitoring_config: |
       app.kubernetes.io/part-of: google-cloud-managed-prometheus
   spec:
     endpoints:
-    - port: prometheus
+    - port: 8404
```

Comment:
Just double-checking that specifying the endpoint this way in configmap directly from haproxy means we can't name the port anymore?